selfdescribe.json

```diff
@@ -556,10 +556,10 @@
                     {
                       "yamlName": "instanceFrom",
                       "doc": "",
-                      "default": "",
+                      "default": null,
```

Comment:
Does it matter that we're moving to null instead of the empty list?

Reply:
That is the default value for a slice in Go. It is functionally equivalent to an empty list.